lib/rubocop/cop/rspec/empty_line_after_example_group.rb

```diff
@@ -31,14 +31,9 @@ class EmptyLineAfterExampleGroup < Cop
 
         def on_block(node)
           return unless example_group?(node)
```

Comment:
Will it work as
```ruby
example_group?(node) do |example_group|
  msg = format(MSG, example_group: example_group.method_name)
  check_missing_separating_lines(example_group, message: msg)
end
```

Reply:
Same above for `EmptyLineAfterExample` and below for `EmptyLineAfterHook`.

Reply:
That would be neat, but we’d have to add a capture in the right place in `RuboCop::RSpec::Language::SelectorSet`. Something to explore in a separate PR, maybe?

Reply:
It still could be
```
example_group?(node) do
  msg = format(MSG, example_group: node.method_name)
  check_missing_separating_lines(node, message: msg)
end
```
but then I would remove the `?` from the name. Still, something that could be explored in a separate PR

Reply:
Up to you. Looks good as is.
There's always an indefinite space for improvement.